datasets.md

```diff
@@ -79,6 +79,10 @@ In an effort to continue diversifying the data being onboarded onto the network,
 | CLEVR Dataset | A Diagnostic Dataset for Compositional Language and Elementary Visual Reasoning. | 19 GB | png&json | https://www.kaggle.com/timoboz/clevr-dataset |
 | COCO2017 | Common Objects in Context | 20 GB | jpg | https://www.kaggle.com/aishwr/coco2017 |
 | COCO 2017 Dataset | COCO is a large-scale object detection, segmentation, and captioning dataset. | 26 GB | jpg | https://www.kaggle.com/awsaf49/coco-2017-dataset |
+| PandaSet Dataset | Public large-scale dataset for autonomous driving |  31GB | Various | https://www.kaggle.com/usharengaraju/pandaset-dataset |
+| AudioSet [Train] | Training audio files for AudioSet |  29GB | Various | https://www.kaggle.com/zfturbo/audioset |
+| GloVe Reddit Comments | Global Vectors for Word Representation based on Reddit comments |  24GB | Various | https://www.kaggle.com/leighplt/glove-reddit-comments |
+| May 2015 Reddit Comments | Get personal with a dataset of comments from May 2015 |  30GB | Various | https://www.kaggle.com/reddit/reddit-comments-may-2015 |
```

Comment:
This feels a little outdated. Seems like a very popular data set but might be good to get a newer one or one with more open usage terms. Currently, the reddit API policy linked states:

> You may not use the Reddit logos and trademark without approval from us
> You may use "for reddit" or "a client for reddit" in the title of your app. You may not use "reddit" without "for" preceding it. You may not use the word "official" in the title, keywords or description in any way that implies the app was developed by Reddit, Inc.
> If your intended usage is commercial, you’ll need approval from us (either by filling out the API terms form or emailing api@reddit.com. Use of the API is considered "commercial" if you are earning money from it, including, but not limited to in-app advertising, in-app purchases or you intend to learn from the data and repackage for sale. Open source use is generally considered non-commercial. 

This could be tough to track for Slingshot teams and I'm not sure where the line is drawn for making income from this dataset for storage providers or clients through Slingshot rewards. Suggest we skip this one for now.

Reply:
Thanks! I will skip it.